packages/devtools/devtools/src/panels/mesh/SignalMessages.tsx

```diff
@@ -104,67 +105,49 @@ const types: ColumnType<SignalResponse>[] = [
       }
     ]
   }
-];
+] as const; // this is ok because getView below will fail typecheck if this array is misdefined
```

Comment:
@mykola-vrmchk check out this use of `as const` to be able to pull types out of the array below as `ViewType`